test/scenarios/attestation/output/src/attestation/azext_attestation/tests/latest/test_FakedTestScenario_scenario.py

```diff
@@ -29,50 +26,44 @@
 
 # Env setup_scenario
 @try_manual
-def setup_scenario(test, rg, rg_2, rg_3):
-    pass
-
-
-# Env mytest
-@try_manual
-def mytest(test, rg, rg_2, rg_3):
-    pass
+def setup_scenario(test, rg, rg_2, rg_3, rg_4):
+    cmd = "az deployment group create --resource-group {{rg}} --template-file \"{}\"".format(os.path.join(TEST_DIR, 'depSto.json'))
+    o = test.cmd(cmd).get_output_in_json()
+    kwargs = {k: v.get("value") for k, v in o.get('properties', {}).get('outputs', {}).items()}
+    test.kwargs.update(kwargs)
 
 
 # Env cleanup_scenario
 @try_manual
-def cleanup_scenario(test, rg, rg_2, rg_3):
+def cleanup_scenario(test, rg, rg_2, rg_3, rg_4):
```

Comment:
Is this rg_4 expected? I actually didn't see any changes of the example, just wondering why